### PR TITLE
Fix for elevation, alias border-radius token, launch button tokens

### PR DESCRIPTION
**Pull request by davyd-akamai**
# Change Summary

## Breaking Changes
- Restructured calendar `background` tokens for improved consistency and flexibility
   - Updated token: `token-component-calendar-selected-item-background-default`
   - Updated token: `token-component-calendar-selected-item-background-hover`

## New Features
- Introduced `inProgress` token for ProgressBar component
- Added `topGradient` token for Appbar component
- Implemented new background tokens:
  - `rangeHover` for date range selection
  - `activeDateHover` for active date hover states
- Created `borderRadius` alias token for unified styling
- Introduced launch button tokens (experimental)
  - Note: These tokens are connected to global styles and may be subject to change as we evaluate their inclusion in CDS 2.0. Dark mode styles are pending.

## Enhancements
- Refined elevation token blur for improved visual depth

### CODE REVIEW

**Comment by davyd-akamai** [comment]
Launch button will be connected to global tokens, since we're still investigating if this button will be used in the CDS 2.0. Dark mode styles are not defined yet

**Comment by jaalah-akamai** [comment]
We're running into an issue parsing the linear gradient. I need to look into this https://github.com/linode/design-language-system/pull/61/files#diff-342f83c57db10e0bec303f072c2b93b04ddade6fc4ba656e182e07dd2cf9202fR770

File: tokens/alias/light.json [hunk 1]
```diff
@@ -1435,6 +1445,13 @@
           "type": "color"
         }
       }
+    },
+    "borderRadius": {
```
Comment (jaalah-akamai) [inline tokens/alias/light.json:1449]: @davyd-akamai We can't have this be the same name as the global token. Maybe we rename this `radius`? Or can we just put this under `border.borderRadius`?

**Comment by davyd-akamai** [comment]
@jaalah-akamai Here will be one more breaking change to a stepper component. UIE team is aware of it.

**Review by jaalah-akamai — APPROVED** [review]
Looks good!

**Pull request merged**